guides/common/modules/con_managing-ostree-content.adoc

```diff
@@ -0,0 +1,21 @@
+[id="con_managing-ostree-content_{context}"]
+= Managing OSTree Content
+
+You can use OSTree to manage bootable, immutable, versioned file system trees.
+
+OSTree makes it easy to install and update Linux-based operating systems on hosts and to switch between versions of operating systems on hosts.
+If a host update fails, you can easily revert the operating system on the host to the previous working version.
+
+You can create an OSTree image using an image builder and expose it in an OSTree repository on an HTTP server.
+Then you can use your {Project} to synchronize and manage OSTree branches from the exposed OSTree repository.
+
+.Prerequisites
+* OSTree content is supported only when running {Project} on RHEL 8.
```

Comment:
```suggestion
* OSTree content is supported only when running {Project} on {RHEL} 8.
```

Does is also work with CentOS 8? See https://docs.theforeman.org/nightly/Installing_Server_on_Red_Hat/index-foreman-el.html#supported-operating-systems_foreman

Reply:
It actually does (confirmed with devs)! Good catch. Please check if you're okay with the condition.